static/index.html

```diff
@@ -0,0 +1,51 @@
+<html>
+  <head>
+    <title>TV shows Vanilla</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div id="page-container">
+      <h1 id="title-1">TV shows APP</h1>
+      <div id="show-information">
+        <img
+          src="/js-react-academy-2024-LovroS02/media/Brooklyn_nine_nine.png"
+          alt="Brooklyn_nine_nine"
+        />
+        <h2 id="show-name">Brooklyn Nine-Nine</h2>
+        <textbox name="Show Description" id="show-description"
```

Comment:
Prvi put se susrećem s ovim tag-om. Nisam siguran da ovaj tag semantički postoji. Kod radi jer je tag "validan" ali nije definiran kao dio HTML konteksta.
Mogao si ovdje staviti i `<nekiTekst>Bla bla bla</nekiTekst>` jer je sintaksa tag-a točna i browser će to renderati kao neki element `nekiTekst` ali semantički on nema nikakvu vrijednost.
Sasvim okej je ovdje staviti `p` tag